src/HealthGPS/risk_factor_adjustable_model.h

```diff
@@ -0,0 +1,60 @@
+#pragma once
+
+#include "HealthGPS.Core/forward_type.h"
+#include "HealthGPS.Core/identifier.h"
+
+#include "map2d.h"
+#include "risk_factor_model.h"
+#include "runtime_context.h"
+
+#include <vector>
+
+namespace hgps {
+
+/// @brief Defines a table type for double values by sex and age
+using RiskFactorSexAgeTable = UnorderedMap2d<core::Gender, core::Identifier, std::vector<double>>;
```

Comment:
I see `sex` is coming here via `core::Gender`, but I don't see `age`. Is it coming via `core::Identifier`?

Reply:
`age` is an integer type, coming directly from `Person.age`. In the case you highlight, the innermost vector element is sized 0-max_age, so we can index it with a plain integer.